tests/ethereum/ethereum.spec.ts

```diff
@@ -0,0 +1,665 @@
+      );
+      const stableDebtToken = await getStableDebtToken(
+        (
+          await hre.deployments.get(STABLE_DEBT_TOKEN_IMPL_ID)
+        ).address
+      );
+
+      await expect(
+        aaveToken.initialize(
+          ZERO_ADDRESS,
+          ZERO_ADDRESS,
+          ZERO_ADDRESS,
+          ZERO_ADDRESS,
+          0,
+          "0",
+          "0",
+          "0x"
+        )
+      ).to.be.revertedWith(InitializeError);
+
+      await expect(
+        delegationToken.initialize(
+          ZERO_ADDRESS,
+          ZERO_ADDRESS,
+          ZERO_ADDRESS,
+          ZERO_ADDRESS,
+          0,
+          "0",
+          "0",
+          "0x"
+        )
+      ).to.be.revertedWith(InitializeError);
+
+      await expect(
+        variableDebtToken.initialize(
+          ZERO_ADDRESS,
+          ZERO_ADDRESS,
+          ZERO_ADDRESS,
+          0,
+          "0",
+          "0",
+          "0x"
+        )
+      ).to.be.revertedWith(InitializeError);
+
+      await expect(
+        stableDebtToken.initialize(
+          ZERO_ADDRESS,
+          ZERO_ADDRESS,
+          ZERO_ADDRESS,
+          0,
+          "0",
+          "0",
+          "0x"
+        )
+      ).to.be.revertedWith(InitializeError);
+    });
+  });
+
+  describe("Paraswap adapters", () => {
+    it("Paraswap Adapters should be deployed", async () => {
+      const paraswapLiquidityAdapter = await hre.deployments.getOrNull(
+        "ParaSwapLiquiditySwapAdapter"
+      );
+      const paraswapRepayAdapter = await hre.deployments.getOrNull(
+        "ParaSwapRepayAdapter"
+      );
+      expect(paraswapLiquidityAdapter).to.not.be.null;
+      expect(paraswapRepayAdapter).to.not.be.null;
+    });
+  });
+
+  describe("Pool config", () => {
+    it(`Should have set the Flash Loan total fees`, async () => {
+      const { FlashLoanPremiums } = poolConfig;
+      const pool = await getPool();
+      const currentPremiumTotal = await pool.FLASHLOAN_PREMIUM_TOTAL();
+      const currentProtocolFee = await pool.FLASHLOAN_PREMIUM_TO_PROTOCOL();
+
+      expect(currentPremiumTotal).equal(FlashLoanPremiums.total);
+      expect(currentProtocolFee).equal(FlashLoanPremiums.protocol);
+    });
+  });
+
+  describe("Governance", () => {
+    it("Governance short executor should be able to list WETH", async () => {
+      const { RateStrategies } = poolConfig;
+
+      const firstStrat = Object.keys(RateStrategies)[0];
+
+      const rateStrategy = await hre.deployments.get(
+        `ReserveStrategy-${firstStrat}`
+      );
+
+      const impersonatedExecutor = await impersonateAddress(
+        ETHEREUM_SHORT_EXECUTOR
+      );
+      const deployer = await getFirstSigner();
+      await deployer.sendTransaction({
+        to: ETHEREUM_SHORT_EXECUTOR,
+        value: oneEther,
+      });
+
+      const poolConfigurator = (await getPoolConfiguratorProxy()).connect(
+        impersonatedExecutor.signer
+      );
+
+      // List WETH for testing purposes
+      await waitForTx(
```

Comment:
We can replace  `waitForTx` for `expect` if we don't need the receipt

Reply:
replaced waitForTx for expect to listen reserve listing event